versions.tf

```diff
@@ -5,7 +5,9 @@
 terraform {
   required_version = ">= 0.12.20, < 0.15"
 
+  # 4.7.0 to 4.9.1 has a security regression: new repositories created via a template have a public visibility
+  # fixed in 4.9.2
   required_providers {
-    github = ">= 2.9, < 4.0, != 3.1.0"
+    github = ">= 4.5, < 5.0, != 4.7.0, != 4.8.0, != 4.9.0, != 4.9.1"
```

Comment:
provider source should be changed to `integrations/github` the old source is deprecated